datafusion/src/execution/mod.rs

```diff
@@ -19,4 +19,7 @@
 
 pub mod context;
 pub mod dataframe_impl;
+pub mod disk_manager;
```

Comment:
Do we need to public this api for the lib?

Reply:
I may need to expose df generated dirs to outer modules, for instance, write shuffle data in df in stage-1 and read shuffle data in reducers run in JVM. I think it's no harm to expose the module out?

Reply:
If we public more API, we should take more effort to maintain the api back compatibility feature.

Reply:
Thanks, I've changed these two into `pub(crate)`